packages/widget/src/controllers/transfers/fungible-token-transfer.ts

```diff
@@ -42,17 +68,25 @@ export class FungibleTokenTransferController implements ReactiveController {
   public destinationNetwork?: Domain;
   public selectedResource?: Resource;
   public resourceAmount: BigNumber = ethers.constants.Zero;
-  public destinationAddress?: string | null = '';
```

Comment:
it's the correct version from https://github.com/sygmaprotocol/sygma-widget/pull/176/files

Reply:
solved